src/metpy/plots/skewt.py

```diff
@@ -593,6 +593,8 @@ def plot_mixing_lines(self, mixing_ratio=None, pressure=None, **kwargs):
         if mixing_ratio is None:
             mixing_ratio = np.array([0.0004, 0.001, 0.002, 0.004, 0.007, 0.01,
                                      0.016, 0.024, 0.032]).reshape(-1, 1)
+        else:
+            mixing_ratio = mixing_ratio.reshape(-1, 1)
```

Comment:
What about instead of adding an `else`, we just make the reshape unconditional:
```suggestion
        if mixing_ratio is None:
            mixing_ratio = np.array([0.0004, 0.001, 0.002, 0.004, 0.007, 0.01,
                                     0.016, 0.024, 0.032])
        mixing_ratio = mixing_ratio.reshape(-1, 1)
```